wicketd-client/src/lib.rs

```diff
@@ -37,6 +37,8 @@ progenitor::generate_api!(
     },
     replace = {
         Duration = std::time::Duration,
+        ProgressEventKindForGenericSpec = update_engine::events::ProgressEventKind<update_engine::GenericSpec<update_engine::NestedError>>,
```

Comment:
This can be `ProgressEventForGenericSpec`.

Reply:
Thanks, fixed in [3d88905](https://github.com/oxidecomputer/omicron/pull/2867/commits/3d889056d828b0b86983e56c8572d3b68aaea4e2)